zarr/creation.py

```diff
@@ -19,32 +22,35 @@
     normalize_storage_path,
     normalize_store_arg,
 )
+from zarr._storage.store import StorageTransformer
+from zarr.sync import Synchronizer
+from zarr.types import ZARR_VERSION, DIMENSION_SEPARATOR, MEMORY_ORDER, MetaArray, PathLike
 from zarr.util import normalize_dimension_separator
 
 
 def create(
-    shape,
-    chunks=True,
-    dtype=None,
+    shape: Union[int, Tuple[int, ...]],
+    chunks: bool = True,
```

Comment:
chunks can also be `Tuple[int, ...]`, so i think the full type should be `Union[Tuple[int, ...], bool]`